Add an entry in glossary for MVP (#42186)
kubernetes/website: content/en/docs/reference/glossary/unknown-version-interoperability-proxy.md

---
title: Unknown Version Interoperability Proxy (UVIP)
id: uvip
date: 2023-07-24
full_link: /docs/concepts/architecture/unknown-version-interoperability-proxy#Enabling-UVIP
short_description: >
  Feature that lets a kube-apiserver proxy a resource request to a different peer API server. 

> aka:

Reviewer: ```suggestion
aka: ["UVIP"]
```

tags:
- architecture
---

> Feature that lets a kube-apiserver proxy a resource request to a different peer API server.

Reviewer: ```suggestion
Feature to let a kube-apiserver proxy a resource request to a peer API server.
```

<!--more-->

> When a cluster has multiple API servers running different versions of Kubernetes, this feature enables resource requests to be served by the correct API server.

Reviewer: ```suggestion
When a cluster has multiple API servers running different versions of Kubernetes, this feature enables resource requests to be served by the correct API server.

UVIP is disabled by default and can be activated by enabling
the feature gate named `UnknownVersionInteroperabilityProxy` when 
{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}} is started.

* Read [UVIP](/docs/concepts/architecture/unknown-version-interoperability-proxy/) for more information.
```

I think adding the Read more will be helpful. Once the PR #42147 is merged , this link will also work WDYT?

Reviewer: Just saw this https://github.com/kubernetes/website/pull/42147#pullrequestreview-1542491937
Well, it will work fine without the link for now It can be added later as well.

Author: Added the suggestion minus the concepts link.
